workflow/envs/gatk_cnv_vcf.yaml

```diff
@@ -0,0 +1,4 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
```

Comment:
should we maybe set a python version? So that we know which version is used to run the script.

Reply:
Yes, that might be good. Should be done everywhere then. What version?